ci-operator/step-registry/load-balancer/load-balancer-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: load-balancer
+  timeout: 120m
+  from: openstack-installer
+  grace_period: 10m
+  commands: load-balancer-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+    - name: CONFIG_TYPE
+      documentation: |
+        The type of config for the environment to deploy.
+
+        * 'minimal' - Configure the install-config with the minimal options to make it work on
+                      our tests, which requires the usage of floating IPs. This config type doesn't
+                      require a proxy server to be deployed.
+        * 'proxy' - Configure the install-config to use a pre-created restricted network (BYON) and
+                   a Cluster Proxy.
+    - name: LB_USER
```

Comment:
Do we have control over what image the LB uses? otherwise, we might just want to hardcode the user to `cloud-user`.

Reply:
yes, this can be achieved by a previous step registry.
However, we could use SHARED_DIR/LB_USER to store it so we don't have this parameter.